app/main.ipynb

```diff
@@ -299,18 +347,26 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 17,
-   "outputs": [],
-   "source": [
-    "# write your code here"
-   ],
+   "source": "result_1 = A[[0, 2, -1]]",
```

Comment:
The output does not match the expected result for Task 5.5. The correct indices should be used to select the rows `[0, 2, 4]` from matrix `A` to get the expected output.